explainer.md

```diff
@@ -913,6 +913,29 @@ xrSession.depthNear = 1.0;
 xrSession.depthFar = 100.0;
 ```
 
+### Handling non-opaque displays
+
+Some devices which support the WebXR Device API may use displays that are not fully opaque, or otherwise show the real world in some capacity. To determine how the display will blend rendered content with the real world, check the `XRSession`'s `worldBlendMode` attribute. It may currently be one of three values, and more may be added in the future if new display technology necessitates it:
+
+  - `opaque`: The world is not visible at all through this display. Transparent pixels in the `baseLayer` will appear black. This is the expected mode for most VR headsets.
```

Comment:
This should be explicit around how an `opaque` world blend mode treats the alpha channel: explicit ignoring it and treating all pixels as if alpha was 1.0.

Reply:
This doesn't sound unreasonable, and I'm going to change the PR text to reflect it because specific is better than non-specific in this case. I would like to verify the behavior of other systems, though. If Daydream, for example. treated this as an alpha-blend against black we'd have to clear the alpha channel to 1.0 prior to compositing in order to comply with this. That's a cost I'd rather not incur if we have a choice.